x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/utils/connector_status_helpers.ts

```diff
@@ -87,9 +88,10 @@ export function connectorStatusToColor(connector: Connector): 'warning' | 'dange
   if (
     isLastSeenOld(connector) ||
     connectorStatus === ConnectorStatus.ERROR ||
-    connector.error === SyncStatus.ERROR ||
-    connector.last_sync_error !== null ||
-    connector.last_access_control_sync_error !== null
+    connector.last_sync_status === SyncStatus.ERROR ||
+    connector.last_access_control_sync_status === SyncStatus.ERROR ||
+    connector.last_sync_error != null ||
```

Comment:
`!==` use `!=` use type coercion when checking for nulls and undefined values (it's seems to be acceptable way for ts)

Reply:
`!= null` is a common thing to check isNullish normally. But I think we have a utility to do the same to prevent misreadings. Just search for `isNullish` in code you may find it and use it if it is useful.